File: 37-spring-boot-DynamicDataSource/src/main/java/com/gzz/user/model/UserCond.java
package com.gzz.user.model;

import java.util.Date;

import com.gzz.common.base.BaseCondition;

import lombok.Getter;
import lombok.Setter;
/**
 * @author https://www.jianshu.com/u/3bd57d5f1074
 * @date 2019-12-24 10:50:00
 */
@Setter
@Getter
public class UserCond extends BaseCondition {

	@Override
	public void addCondition() {
	}

	private Integer id;
	private String name;
	private Date birthday;

}

File: 99-spring-boot-kotlin/src/main/java/com/gzz/mvc/UserCond.kt
package com.gzz.mvc

import com.gzz.base.BaseCondition

data class UserCond(
		var id: Long?,
		var password: String?,
		var name: String?,
		var login_id: String?
) : BaseCondition() {
	override fun addCondition() {
		add(name, " and name=?");
		add(id, " and id=?");
	}
}
 
 
